playbooks/roles/hammer_devel/tasks/hammer_config.yml

```diff
@@ -9,31 +9,43 @@
   command: cp ~/hammer-cli/config/cli_config.template.yml ~/.hammer/cli_config.yml
 
 - name: 'Configure hammer-cli-foreman'
-  blockinfile:
+  command: cp ~/hammer-cli-foreman/config/foreman.yml ~/.hammer/cli.modules.d/
+
+- name: 'Update hammer-cli-foreman host'
+  lineinfile:
+    dest: ~/.hammer/cli.modules.d/foreman.yml
+    line: '  :host: {{ hammer_devel_host }}'
+    regexp: 'host'
+
+- name: 'Update hammer-cli-foreman username'
+  lineinfile:
+    dest: ~/.hammer/cli.modules.d/foreman.yml
+    line: '  :username: {{ hammer_devel_username }}'
+    regexp: 'username'
+
+- name: 'Update hammer-cli-foreman password'
+  lineinfile:
     dest: ~/.hammer/cli.modules.d/foreman.yml
-    create: true
-    block: |
-      :foreman:
-        :enable_module: true
-        :host: {{ hammer_devel_host }}
-        :username: {{ hammer_devel_username }}
-        :password: {{ hammer_devel_password }}
+    line: '  :password: {{ hammer_devel_password }}'
+    regexp: 'password'
+
+- name: 'Enable hammer-cli-foreman-admin'
+  command: cp ~/hammer-cli-foreman-admin/config/foreman_admin.yml ~/.hammer/cli.modules.d/
+
+- name: 'Configure hammer-cli-foreman-admin logging core'
+  command: cp ~/hammer-cli-foreman-admin/config/cli.modules.d/foreman_admin_logging_core.yml ~/.hammer/cli.modules.d/
+
+- name: 'Configure hammer-cli-foreman-admin logging for katello'
+  command: cp ~/hammer-cli-foreman-admin/config/cli.modules.d/foreman_admin_logging_katello.yml ~/.hammer/cli.modules.d/
```

Comment:
Yeah, I was thinking about what happens if they change the file (i.e. from foreman.yml to foreman.yml.example, how it should be...) and I'd rather Forklift start failing from not finding the source file than from not configuring hammer properly (hammer wouldn't recognize example files, for instance) - hard fail vs. soft fail.